Configuration/Geometry/python/GeometryRecoDB_cff.py

```diff
@@ -17,8 +17,7 @@
 from Geometry.CaloEventSetup.EcalTrigTowerConstituents_cfi import *
 from Geometry.EcalMapping.EcalMapping_cfi import *
 from Geometry.EcalMapping.EcalMappingRecord_cfi import *
-from Geometry.HcalCommonData.hcalDDDSimConstants_cfi import *
-from Geometry.HcalCommonData.hcalDDDRecConstants_cfi import *
+from Geometry.HcalCommonData.hcalDBConstants_cff import *
```

Comment:
`hcalDBConstants_cff.py` simply contains the two lines that are being replaced, so there should be no change in behavior.